src/components/Greetings/greetings.js

```diff
@@ -0,0 +1,29 @@
+import "./greeting.css";
+
+const toGreet = (lang) => {
+    let greet = ""
```

Comment:
```suggestion
    let greet;
```

Reply:
You already have a default string "hello", you wouldn't need to give a value to the variable